test/machineslib.py

```diff
@@ -410,7 +410,7 @@ def stop_all():
                       '    echo "$out" | grep -q "domain is not running"; '
                       "  fi; done")
             m.execute("runuser -l admin -c 'for d in $(virsh -c qemu:///session list --all --name); do "
-                      "virsh -c qemu:///session undefine $d; done'")
+                      "virsh -c qemu:///session undefine $d --snapshots-metadata; done'")
```

Comment:
Doesn't the system session need this to then?

Reply:
Apparently not.  For the system connection we use `virsh destroy`, but for the session connection we do `virsh undefine`. I am not sure why the `undefine` is needed. `/home/admin/.local/share/libvirt/` is restored after each test, doesn't that make the session machines go away? *shrug*

Reply:
I'm going to leave this for an expert :)